src/pam/error.rs

```diff
@@ -201,7 +201,19 @@ impl fmt::Display for PamError {
         match self {
             PamError::UnexpectedNulByte(_) => write!(f, "Unexpected nul byte in input"),
             PamError::Utf8Error(_) => write!(f, "Could not read input data as UTF-8 string"),
-            PamError::Pam(tp) => write!(f, "PAM returned an error ({tp:?}): {}", tp.get_err_msg()),
+            PamError::Pam(PamErrorType::AuthError) => {
+                write!(f, "account validation failure, is your account locked?")
```

Comment:
```suggestion
                write!(f, "Account validation failure, is your account locked?")
```
You know my finger is really itching to hit "approve" when I start nitpicking about the inconsistent capitalization of this error message. :-)